great_expectations/data_context/store/checkpoint_store.py

```diff
@@ -293,3 +293,11 @@ def _add(self, key: DataContextKey, value: V1Checkpoint, **kwargs):
         if not self.cloud_mode:
             value.id = str(uuid.uuid4())
         return super()._add(key=key, value=value, **kwargs)
+
+    @override
+    def _update(self, key: DataContextKey, value: V1Checkpoint, **kwargs):
+        try:
+            super()._update(key=key, value=value, **kwargs)
+        except gx_exceptions.StoreBackendError as e:
+            name = key.to_tuple()[0]
+            raise ValueError(f"Could not find existing Checkpoint '{name}' to update") from e
```

Comment:
isn't this the same issue as before? There are other valid errors -- such as validation --  Cloud backend can throw, so this error message is too specific.